Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/Mesh/MeshComponentBus.h

```diff
@@ -106,6 +106,10 @@ namespace AZ
             : public ComponentBus
         {
         public:
+            // Notifications can be triggered from job threads, so this uses a mutex to guard against
+            // listeners joining or leaving the ebus on other threads mid-notification.
+            typedef AZStd::recursive_mutex MutexType;
```

Comment:
using is preferred over typedef
```suggestion
            // Notifications can be triggered from job threads, so this uses a mutex to guard against
            // listeners joining or leaving the ebus on other threads mid-notification.
            using MutexType = AZStd::recursive_mutex;
```